conf/messages.en

```diff
@@ -1265,9 +1265,9 @@ page.iht.application.gifts.lastYears.givenAway.p1 = You need to find out what gi
 page.iht.application.gifts.lastYears.givenAway.p2 = The tax years for this estate are:
 
 page.iht.application.giftsDetails.browserTitle = Value of gifts for year
-page.iht.application.giftsDetails.subtitle = Gifts and exemptions for tax year {0} to {1}
-page.iht.application.giftsDetails.value.label = Total value of gifts
-page.iht.application.giftsDetails.exemptions.label = Total value of exemptions
+page.iht.application.giftsDetails.subtitle = {0} to {1}
```

Comment:
![image](https://user-images.githubusercontent.com/14074579/27431102-2881f5be-5743-11e7-9aff-53273ae97593.png)
Could probably do with looking at putting the two dates on separate lines to assist in readability